recipes/clove-unit/all/conanfile.py

```diff
@@ -40,11 +41,13 @@ def package_id(self):
         self.info.clear()
 
     def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "CloveUnit")
+        self.cpp_info.set_property("cmake_target_name", "CloveUnit::CloveUnit")
         # Folders not used for header-only
         self.cpp_info.bindirs = []
         self.cpp_info.libdirs = []
         # TODO: to remove in conan v2 once cmake_find_package_* generators removed
-        self.cpp_info.filenames["cmake_find_package"] = "CloveUnit"
-        self.cpp_info.filenames["cmake_find_package_multi"] = "CloveUnit"
-        self.cpp_info.names["cmake_find_package"] = "CloveUnit"
-        self.cpp_info.names["cmake_find_package_multi"] = "CloveUnit"
+        # self.cpp_info.filenames["cmake_find_package"] = "CloveUnit"
```

Comment:
why not remove dead code altogether?

Reply:
I was trying possible combinations to make CI pass for both v1 and v2